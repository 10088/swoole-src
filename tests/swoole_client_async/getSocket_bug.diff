001+ Fatal error: Uncaught Error: Call to undefined method swoole_client::getSocket() in /src/swoole-src/tests/swoole_client_async/getSocket_bug.php:13
001- SUCCESS
002+ Stack trace:
003+ #0 [internal function]: {closure}(Object(swoole_client))
004+ #1 /src/swoole-src/tests/swoole_client_async/getSocket_bug.php(30): swoole_event::wait()
005+ #2 {main}
006+   thrown in /src/swoole-src/tests/swoole_client_async/getSocket_bug.php on line 13