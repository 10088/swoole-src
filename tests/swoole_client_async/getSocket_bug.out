Fatal error: Uncaught Error: Call to undefined method swoole_client::getSocket() in /src/swoole-src/tests/swoole_client_async/getSocket_bug.php:13
Stack trace:
#0 [internal function]: {closure}(Object(swoole_client))
#1 /src/swoole-src/tests/swoole_client_async/getSocket_bug.php(30): swoole_event::wait()
#2 {main}
  thrown in /src/swoole-src/tests/swoole_client_async/getSocket_bug.php on line 13